### PR TITLE
v2.2.0

### PR DESCRIPTION
**Pull request by vladikoff**
@wondersloth looks good?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -287,70 +299,70 @@ grunt.initConfig({
...
         specs: 'spec/*Spec.js',
-        helpers: 'spec/*Helper.js',
-        template: 'custom.tmpl',
-        sandboxArgs: {
-          args: ['--no-sandbox'],
-          timeout: 3000,
-          defaultViewport: {
-            isMobile: true
-          }
-        }
```
Comment (steveoh) [inline README.md:353]: i don't think you should remove the sandbox args from the sandbox arg demo ;)

Comment (steveoh) [reply README.md:353]: oh i see you moved them to an examples file?

Comment (vladikoff) [reply README.md:353]: yeap!

**Comment by steveoh** [comment]
https://github.com/gruntjs/grunt-contrib-jasmine#release-history should get an update

**Comment by vladikoff** [comment]
oh weird, thought it did!

**Pull request merged**